spec/controllers/admin/offers_controller_spec.rb

```diff
@@ -297,16 +294,14 @@
           Fabricate(
             :offer,
             offer_type: 'retail',
-            insurance_dollars: 10,
-            insurance_percent_whole: 12
+            insurance_info: "1000 or best offer"
           )
         put :update,
             params: {
               id: retail_offer.id,
-              offer: { insurance_dollars: nil, insurance_percent_whole: nil }
+              offer: { insurance_info: nil }
             }
-        expect(retail_offer.reload.insurance_cents).to eq nil
-        expect(retail_offer.reload.insurance_percent_whole).to eq nil
+        expect(retail_offer.reload.insurance_info).to eq ""
```

Comment:
I wasn't expecting to have to change the right side of this assertion (from `.to eq nil` to `.to eq ""`). Maybe a reviewer can tell me why I would have had to do this?